docs/changes/2315.irf-maker.rst

```diff
@@ -0,0 +1 @@
+Add a `make-irf tool` able to produce irfs given a gamma, proton and electron DL2 input files.
```

Comment:
This file needs to be removed

Reply:
I only see two change log entries here. The other one uses the correct PR number and topic